aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -587,7 +589,7 @@ bool can_use_flash_attention(sdp_params const& params, bool debug) {
   }
   if (has_only_dense_inputs(params)) {
     constexpr auto dense_constraints = array_of<bool (*)(sdp_params const&, bool)>(
-        check_batch_size_and_num_heads_dense,
+        check_batch_size_and_num_heads_dense<true>, // supports_grouped_query_attention = true,
```

Comment:
super nit: I think you can just inline the kwarg name similiar to the line below for ignore_singelton_dim